1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
+- Public agent nodes are used primarily for externally facing reverse proxy load balancers, like Marathon-LB.
+- Clusters usually only have a few public agent nodes, because a single load balancer can handle proxying multiple services.
+- For more information, see [Converting Agent Node Types](/docs/1.8/administration/installing/custom/convert-agent-type/).
 
-# <a name="exhibitorforzookeeper"></a>Exhibitor for ZooKeeper
+### <a name="bootstrap-machine"></a>Bootstrap Machine
 
-DC/OS uses ZooKeeper, a high-performance coordination service to manage the installed DC/OS services. Exhibitor automatically configures your ZooKeeper installation on the master nodes during your DC/OS installation.
+A bootstrap machine is the machine on which the DC/OS installer artifacts are configured, built, and distributed.
 
-# <a name="framework"></a>Framework
+- The bootstrap machine is not technically considered part of the cluster, in that it does not have DC/OS installed on it (this may change in the future), but it does need to be accessible to/from (depending on install method) the machines in the cluster via infrastructure networking in order to perform remote installation.
+- The bootstrap machine is sometimes used as a jumpbox to control SSH access into other nodes in the cluster for added security/logging.
+- One method of allowing master nodes to change IPs involves running Exhibitor/ZooKeeper on the bootstrap machine. Other alternatives include using S3, DNS, or static IPs, with various tradeoffs.
+- If not required for managing master node IP changes or as an SSH jumpbox, the bootstrap machine may be shut down after bootstrapping and spun up on demand when new nodes need to be added to the cluster.
+- For more information, see the [system requirements](/docs/1.8/administration/installing/custom/system-requirements/#bootstrap-node).
 
-A Mesos framework is the combination of a Mesos scheduler and an optional custom executor. A framework receives resource offers describing CPU, RAM, etc. from the leading Mesos master, and allocates them for discrete tasks that can be launched on Mesos agent nodes. Mesosphere-certified Mesos frameworks, called DC/OS services, are packaged and available from public [GitHub package repositories][5]. DC/OS services include Mesosphere-certified Mesos frameworks and other applications.
+### <a name="service"></a>Service
 
-# <a name="master"></a>Master
+A DC/OS service is a set of one or more service instances that can be started and stopped as a group and restarted automatically if they exit before being stopped.
 
-A Mesos master aggregates resource offers from all [agent nodes][8] and provides them to registered frameworks. For more details about the Mesos master, read about [Mesos Master Configuration][9].
+- Services is currently just a UI abstraction that translates to Marathon apps and pods in the CLI and API. This distinction will change over time as the name "service" is pushed upstream into component APIs.
```

Comment:
Yes, that's how we currently define it here dcos.io/docs/1.8/usage/webinterface/